### PR TITLE
Article for hacktoberfest 2023 participation.

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Hacktoberfest 2023 article

### CODE REVIEW

**Comment by RanabirChakraborty** [comment]
Still, a few links left to be updated. Once I get those will update it.

@bstansberry @Skyllarr @fl4via @marcosgopen @iweiss please review this PR.

File: _posts/2023-09-26-Hacktoberfest-2023.adoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+* *Contribute:* Select an issue that matches your interest and start working on it. You'll have the option to receive a Hacktoberfest t-shirt or have a tree planted in your name as a token of appreciation.
+
+== Get started with our projects
+
+This list contains some really interesting projects that are under the WildFly umbrella, waiting for your magic touch. There’s something for everyone from Java and Ansible to TypeScript and JavaScript.
+
+. *WildFly:* WildFly is a powerful, modular, and lightweight application server that empowers you to build amazing applications.
+* Here is the https://github.com/wildfly/wildfly[GitHub repository], https://github.com/wildfly/wildfly/blob/main/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/browse/WFLY-17539?filter=12421535[Hacktoberfest Issues link].
+
+. *WildFly Elytron:* WildFly Elytron is a new WildFly sub-project that is completely replacing the combination of PicketBox and JAAS as the WildFly client and server security mechanism.
```
Comment (bstansberry) [inline _posts/2023-09-26-Hacktoberfest-2023.adoc:35]: This needs to be updated, as Elytron is not new now and the fact they replaced Picketbox is no longer a key thing to mention.

File: _posts/2023-09-26-Hacktoberfest-2023.adoc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* *Follow the Project guidelines:* Read the respective Contributing Guide, Code of Conduct, and follow best practices like forking the repository, creating a new branch, and making a meaningful contribution.
+* *Contribute:* Select an issue that matches your interest and start working on it. You'll have the option to receive a Hacktoberfest t-shirt or have a tree planted in your name as a token of appreciation.
+
+== Get started with our projects
+
+This list contains some really interesting projects that are under the WildFly umbrella, waiting for your magic touch. There’s something for everyone from Java and Ansible to TypeScript and JavaScript.
+
+. *WildFly:* WildFly is a powerful, modular, and lightweight application server that empowers you to build amazing applications.
+* Here is the https://github.com/wildfly/wildfly[GitHub repository], https://github.com/wildfly/wildfly/blob/main/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/browse/WFLY-17539?filter=12421535[Hacktoberfest Issues link].
```
Comment (bstansberry) [inline _posts/2023-09-26-Hacktoberfest-2023.adoc:33]: s/Here is/Here are/g

File: _posts/2023-09-26-Hacktoberfest-2023.adoc [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+. *WildFly Elytron:* WildFly Elytron is a new WildFly sub-project that is completely replacing the combination of PicketBox and JAAS as the WildFly client and server security mechanism.
+* Here is the https://github.com/wildfly-security/wildfly-elytron[GitHub repository], https://github.com/wildfly-security/wildfly-elytron/blob/2.x/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/browse/ELY-2636?filter=12383825[Hacktoberfest Issues link].
```
Comment (bstansberry) [inline _posts/2023-09-26-Hacktoberfest-2023.adoc:36]: s/Here is/Here are/g

File: _posts/2023-09-26-Hacktoberfest-2023.adoc [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+. *Ansible Middleware:* Ansible Middleware is a suite of supported content that combines the power of Ansible automation with Red Hat's Middleware portfolio. It allows organizations to meet their business demands and automate the installation, configuration, and management of Enterprise Middleware.
+* Here is the https://github.com/ansible-middleware[GitHub repositories], https://github.com/ansible-middleware/keycloak/blob/main/CONTRIBUTING.md[Contributing Guide], and https://github.com/search?q=org%3Aansible-middleware+state%3Aopen+label%3Ahacktoberfest&type=issues[Hacktoberfest Issues link].
```
Comment (bstansberry) [inline _posts/2023-09-26-Hacktoberfest-2023.adoc:39]: s/Here is/Here are/g

File: _posts/2023-09-26-Hacktoberfest-2023.adoc [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+. *Undertow:* Undertow is a versatile and high-performance web server that provides both blocking and non-blocking APIs based on NIO. Its modular architecture allows you to build a web server tailored to your needs, from a full Java EE servlet 4.0 container to a low-level non-blocking handler.
+* Here is the https://github.com/undertow-io/undertow[GitHub repository], https://github.com/undertow-io/undertow/blob/master/CONTRIBUTING.md[Contributing Guide], and Hacktoberfest Issues link.
+
+. *Narayana:* Narayana is a transactions toolkit that provides support for applications using various standards-based transaction protocols. It's an essential component of the WildFly application server and continues to be developed as a standalone transaction manager.
+* Here is the https://github.com/jbosstm/narayana[GitHub repository], https://github.com/jbosstm/narayana/blob/main/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/browse/JBTM-2493?filter=12421681[Hacktoberfest Issues link].
```
Comment (bstansberry) [inline _posts/2023-09-26-Hacktoberfest-2023.adoc:45]: s/Here is/Here are/g

File: _posts/2023-09-26-Hacktoberfest-2023.adoc [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+. *Infinispan Console:* Infinispan Console is a front-end application for the web console of the new Infinispan Server. Built using Patternfly 5 and React 18, it's a dynamic and engaging project to contribute to.
```
Comment (bstansberry) [inline _posts/2023-09-26-Hacktoberfest-2023.adoc:47]: s/new Infinispan/Infinispan

Per https://github.com/infinispan/infinispan-console/releases/tag/0.1.Final this dates back to late 2019.

File: _posts/2023-09-26-Hacktoberfest-2023.adoc [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+* Here is the https://github.com/infinispan/infinispan-console[GitHub repository], Contributing Guide, and https://github.com/infinispan/infinispan-console/issues[Hacktoberfest Issues link].
```
Comment (bstansberry) [inline _posts/2023-09-26-Hacktoberfest-2023.adoc:48]: s/Here is/Here are/g

File: _posts/2023-09-26-Hacktoberfest-2023.adoc [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+* Here is the https://github.com/wildfly-security/wildfly-elytron[GitHub repository], https://github.com/wildfly-security/wildfly-elytron/blob/2.x/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/browse/ELY-2636?filter=12383825[Hacktoberfest Issues link].
+
+. *Ansible Middleware:* Ansible Middleware is a suite of supported content that combines the power of Ansible automation with Red Hat's Middleware portfolio. It allows organizations to meet their business demands and automate the installation, configuration, and management of Enterprise Middleware.
+* Here is the https://github.com/ansible-middleware[GitHub repositories], https://github.com/ansible-middleware/keycloak/blob/main/CONTRIBUTING.md[Contributing Guide], and https://github.com/search?q=org%3Aansible-middleware+state%3Aopen+label%3Ahacktoberfest&type=issues[Hacktoberfest Issues link].
+
+. *Undertow:* Undertow is a versatile and high-performance web server that provides both blocking and non-blocking APIs based on NIO. Its modular architecture allows you to build a web server tailored to your needs, from a full Java EE servlet 4.0 container to a low-level non-blocking handler.
```
Comment (bstansberry) [inline _posts/2023-09-26-Hacktoberfest-2023.adoc:41]: s/Java EE servlet 4.0/Jakarta Servlet/g

File: _posts/2023-09-26-Hacktoberfest-2023.adoc [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+Hacktoberfest is a month-long event in October that encourages Open Source contributions and developers worldwide to give back to the global tech community. It's a fantastic opportunity for the contributors to showcase their skills and get to know about other Open Source communities.
+
+To participate you simply need to:
+
+
+* *Register:* https://hacktoberfest.com/auth/[Sign up] on the official Hacktoberfest website to get started.
+* *Discover Projects:* Explore the Hacktoberfest website to find a project that sparks your interest. You can also check out the curated list of projects we've prepared for you.
+* *Follow the Project guidelines:* Read the respective Contributing Guide, Code of Conduct, and follow best practices like forking the repository, creating a new branch, and making a meaningful contribution.
+* *Contribute:* Select an issue that matches your interest and start working on it. You'll have the option to receive a Hacktoberfest t-shirt or have a tree planted in your name as a token of appreciation.
```
Comment (dpanshug) [inline _posts/2023-09-26-Hacktoberfest-2023.adoc:26]: This year they are not gifting any t-shirts :( 
https://hacktoberfest.com/participation/

File: _posts/2023-09-26-Hacktoberfest-2023.adoc [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+
+== Get started with our projects
+
+This list contains some really interesting projects that are under the WildFly umbrella, waiting for your magic touch. There’s something for everyone from Java and Ansible to TypeScript and JavaScript.
```
Comment (dpanshug) [inline _posts/2023-09-26-Hacktoberfest-2023.adoc:30]: I'm sceptical about placing Infinispan console under Wildfy umbrella. Though Infinispan has some close work with Wildly, console is not the one. 

Should we add new heading or keep it as it is?? :thinking:

File: _posts/2023-09-26-Hacktoberfest-2023.adoc [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+* Here is the https://github.com/jbosstm/narayana[GitHub repository], https://github.com/jbosstm/narayana/blob/main/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/browse/JBTM-2493?filter=12421681[Hacktoberfest Issues link].
+
+. *Infinispan Console:* Infinispan Console is a front-end application for the web console of the new Infinispan Server. Built using Patternfly 5 and React 18, it's a dynamic and engaging project to contribute to.
+* Here is the https://github.com/infinispan/infinispan-console[GitHub repository], Contributing Guide, and https://github.com/infinispan/infinispan-console/issues[Hacktoberfest Issues link].
+
+== How to reach out to us for Hacktoberfest related queries?
+
+Please join our https://wildfly.zulipchat.com/#narrow/stream/Hacktoberfest[Hacktoberfest Zulip chatroom] and reach out to us if you have any queries or need assistance. We’re here to ensure a smooth journey for you.
```
Comment (dpanshug) [inline _posts/2023-09-26-Hacktoberfest-2023.adoc:52]: Can we point separate chat room for Infinispan console https://infinispan.zulipchat.com/#narrow/stream/118645-infinispan/topic/hacktoberfest

File: _posts/2023-09-26-Hacktoberfest-2023.adoc (hunk 6 above)
Comment (dpanshug) [inline _posts/2023-09-26-Hacktoberfest-2023.adoc:47]: s/project to/project for React developers to/g

**Review by dpanshug — CHANGES_REQUESTED** [review]

File: _posts/2023-09-26-Hacktoberfest-2023.adoc (hunk 9 above)
Comment (RanabirChakraborty) [reply _posts/2023-09-26-Hacktoberfest-2023.adoc:26]: Ah, nice catch. I'll update it.

File: _posts/2023-09-26-Hacktoberfest-2023.adoc (hunk 10 above)
Comment (RanabirChakraborty) [reply _posts/2023-09-26-Hacktoberfest-2023.adoc:30]: As the article will be published from wildfly.org we are considering the projects from the WildFly umbrella only. Adding a new heading will not be a good idea. Let me know your thoughts.

File: _posts/2023-09-26-Hacktoberfest-2023.adoc (hunk 11 above)
Comment (RanabirChakraborty) [reply _posts/2023-09-26-Hacktoberfest-2023.adoc:52]: Currently, we are asking contributors to join a single chatroom as it'll be easy plus Zulip has a feature to create new threads so if needed contributors can create project-specific new threads and discuss their queries.

File: _posts/2023-09-26-Hacktoberfest-2023.adoc (hunk 1 above)
Comment (RanabirChakraborty) [reply _posts/2023-09-26-Hacktoberfest-2023.adoc:35]: @Skyllarr can you please provide me a description for WildFly Elytron?

File: _posts/2023-09-26-Hacktoberfest-2023.adoc (hunk 10 above)
Comment (dpanshug) [reply _posts/2023-09-26-Hacktoberfest-2023.adoc:30]: np, it wil work too :+1:

File: _posts/2023-09-26-Hacktoberfest-2023.adoc (hunk 11 above)
Comment (RanabirChakraborty) [reply _posts/2023-09-26-Hacktoberfest-2023.adoc:52]: We all can join the chatroom to drive the initiative together, providing separate chatrooms for every project will make it confusing.

Comment (bstansberry) [reply _posts/2023-09-26-Hacktoberfest-2023.adoc:52]: Unless @dpanshug believes people from Infinispan are committed to monitoring WildFly zulip, we shouldn't tell Infinispan contributors to go there. It's a completely different community. The WildFly developers are unlikely to have any idea about Infinispan Console. WF does not use it. There are only a couple of us who are familiar with contributing to any part of Infinispan, so even simple process-oriented questions may go unaddressed if we're relying on WildFly devs for coverage.

Comment (RanabirChakraborty) [reply _posts/2023-09-26-Hacktoberfest-2023.adoc:52]: After having a discussion with @dpanshug he decided not to go with the article as WildFly and its components are not much aligned with the Infinispan console and both the communities are completely different. 
So I'll remove the Infinispan console part.

File: _posts/2023-09-26-Hacktoberfest-2023.adoc (hunk 1 above)
Comment (Skyllarr) [reply _posts/2023-09-26-Hacktoberfest-2023.adoc:35]: @RanabirChakraborty WildFly Elytron is a set of APIs and SPIs for application server security and client side security. Within the WildFly server, WildFly Elytron is used to secure applications deployed to the server as well as management access to the server. It is a standalone library that can also be used in other Java server environments.

File: _posts/2023-09-26-Hacktoberfest-2023.adoc [hunk 12]
```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "Hacktoberfest 2023"
+date:   2023-09-26
```
Comment (Skyllarr) [inline _posts/2023-09-26-Hacktoberfest-2023.adoc:4]: Just a minor, the date should be updated now

**Comment by RanabirChakraborty** [comment]
@bstansberry updated the review comments accordingly. Can you please review it for the last time :)

**Review by bstansberry — APPROVED** [review]

**Pull request merged**